esp-link/cgiflash.c

```diff
@@ -150,10 +148,10 @@ static ETSTimer flash_reboot_timer;
 int ICACHE_FLASH_ATTR cgiRebootFirmware(HttpdConnData *connData) {
 	if (connData->conn==NULL) return HTTPD_CGI_DONE; // Connection aborted. Clean up.
 
-        if (!canOTA()) {
-          errorResponse(connData, 400, flash_too_small);
-          return HTTPD_CGI_DONE;
-        }
+  if (!canOTA()) {
+    errorResponse(connData, 400, flash_too_small);
+    return HTTPD_CGI_DONE;
+  }
```

Comment:
Actually that whole file is 4 space indented rather then 2.  You want it 4 space indented or normalized to 2 space indention?